framework/doc/content/source/problems/DumpObjectsProblem.md

```diff
@@ -12,7 +12,7 @@ the screen and Moose will halt execution.
 
 ## Example
 
-The input file `two_block_new.i` is a test for the TensorMechanics master action, an action 
+The input file `two_block_new.i` is a test for the TensorMechanics primary action, an action 
```

Comment:
Ugh, that "Master" action is actually my baby...

Reply:
So we have some decisions to make. I decided against removing the [mM]aster language from multiapps because *I think* (it's been a hazy last 24 hours) it was going to required me to regold a fair amount of stuff (and now I'm wondering if it would have just been `git mv'ing` which would have been ok). And within the last 30 minutes I did a massive emacs query tags replace of Master, and chose *not* to rename the Master actions. Probably because I'm getting tired and I'm not sure how I would deprecate a Syntax gracefully (maybe it exists; I haven't researched it).

Of course if I don't rename the Master actions, I also need to go back and undo some blind "master -> primary" replacements.

I dunno man this is tough. My feeling at this point is that master in a vacuum is ok, and I'm also ok with master/sub(ordinate).

The slave replacements were easy, but the master replacements are hard and time consuming. I'm losing steam

Reply:
Yeah, I feel you. I think the Master action is by far not as problematic as the master in the master/slave relationship. I suggest focussing on the contact alone for now.